compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAssignNpuDmaBdIds.cpp

```diff
@@ -160,6 +159,34 @@ LogicalResult assignNpuDmaBdIds(AMDAIE::WorkgroupOp workgroupOp) {
             shimTileToGeneratorMap[tileOp.getResult()];
         uint32_t value = bdIdOp.getValue();
         generator.releaseBdId(value);
+        // TODO(zhewen): This is a temporary solution to make any BD ID wrap
```

Comment:
I was only allowing the chaining if `next_bd > current_bd`, so I may skip some values (by wrapping earlier) to avoid a short chain. However, such a constraint as well as this piece of code has been removed,  after refactoring the BD ID chaining pass.